opta/commands/deploy.py

```diff
@@ -18,7 +18,7 @@
 
 @click.command()
 @click.option(
-    "-i", "--image", required=True, help="Your local image in the for myimage:tag"
+    "-i", "--image", help="Your local image in the for myimage:tag", default=None
```

Comment:
now that this is optional let's also update the doc to reflect this and add an example https://github.com/run-x/opta/blob/d8eb1a3c78f8c58f267ff68af66b97b8b7ca6677/opta/commands/deploy.py#L63-L76